logger.go

```diff
@@ -58,10 +62,18 @@ func Init(name string, verbose, systemLog bool, logFile io.Writer) {
 	}
 
 	flags := log.Ldate | log.Lmicroseconds | log.Lshortfile
-	infoLog = log.New(io.MultiWriter(iLogs...), "INFO: ", flags)
-	errorLog = log.New(io.MultiWriter(eLogs...), "ERROR: ", flags)
-	fatalLog = log.New(io.MultiWriter(eLogs...), "FATAL: ", flags)
-	initialized = true
+
+	var l logger
+	l.infoLog = log.New(io.MultiWriter(iLogs...), "INFO: ", flags)
+	l.errorLog = log.New(io.MultiWriter(eLogs...), "ERROR: ", flags)
+	l.fatalLog = log.New(io.MultiWriter(eLogs...), "FATAL: ", flags)
+
+	if !initialized {
```

Comment:
Given that Init() will be called infrequently I think it would suffice to just always Lock(), defer Unlock() and check against defaultLogger itself, dropping initialized entirely?